DOCSP-29504-many-to-one-clusters (#161)
mongodb/docs-cluster-to-cluster-sync: source/includes/opts/migrationName.rst

.. reference/configuration.txt
.. reference/mongosync.txt

.. versionadded:: 1.7.0

> Sets a migration name for a sync operation. For example, you can set a

Reviewer: I would remove the for example here, there are 3x "For examples" currently for this one parameter section.

```suggestion
Sets a migration name for a sync operation. You can set a
```

Author: Removed the other "for example" in the middle. That works a bit better, otherwise the user might think the migration name is the main reason for the name.

> migration name to identify the sync operations from multiple source
> clusters into one destination cluster.

Reviewer 2: ```suggestion
migration name to identify each sync operation from multiple source
clusters into one destination cluster.
```

Author: Done.

> The ``migrationName`` string can contain up to 44 alphanumeric and

Reviewer: It looks like migrationName also supports spaces, should we call this out in the description?

```suggestion
The ``migrationName`` string can contain up to 44 alphanumeric, space, and
```

Author: Cannot have spaces. Updated text.

and underscore characters. ``migrationName`` is appended to the string

> ``"mongosync_internal_"``.

Reviewer 2: ```suggestion
``"mongosync_internal_"`` to serve as the migration metadata database name.
```

If you set ``migrationName`` to

> ``"cluster_27000_to_cluster_35000_sync"``, the resulting final string is

Reviewer 2: ```suggestion
``"cluster_27000_to_cluster_35000_sync"``, the resulting mongosync metadata database name is
```

> For example, if you set ``migrationName`` to ``"Cluster 27000 to cluster
> 35000 sync"``, the final string is ``"mongosync_internal_Cluster 27000
> to cluster 35000 sync"``.

Reviewer: ```suggestion
If you specify ``migrationName`` as ``Cluster 27000 to cluster 35000 sync`` the resulting final string is ``mongosync_internal_Cluster 27000 to cluster 35000 sync``.
```

Author: Done.
